gamemodes/terrortown/entities/weapons/weapon_zm_improvised.lua

```diff
@@ -10,11 +10,11 @@ if SERVER then
 
 	---
 	-- @realm server
-	cv_crowbar_unlocks = CreateConVar("ttt_crowbar_unlocks", "1", FCVAR_ARCHIVE)
+	cv_crowbar_unlocks = CreateConVar("ttt_crowbar_unlocks", "1", {FCVAR_ARCHIVE, FCVAR_NOTIFY})
 
 	---
 	-- @realm server
-	cv_crowbar_pushforce = CreateConVar("ttt_crowbar_pushforce", "395", FCVAR_NOTIFY)
+	cv_crowbar_pushforce = CreateConVar("ttt_crowbar_pushforce", "395", {FCVAR_ARCHIVE, FCVAR_NOTIFY})
```

Comment:
What about including those two?

Reply:
I don't understand what you mean

Reply:
They aren't included in the menu are they?

Reply:
you are absolutely right, I will add them as well

Reply:
done